src/overlays/actors/ovl_En_Ma4/z_en_ma4.c

```diff
@@ -690,7 +690,7 @@ void EnMa4_InitHorsebackGame(EnMa4* this, PlayState* play) {
 
```

Comment:
Not really apart of the overall PR, but on line 606 in this file there is a `!(CHECK_QUEST_ITEM(QUEST_SONG_EPONA))` that could remove a set of parens.